app/src/docs/components/DocsPages/API/UsingStreamsViaApi/index.jsx

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import DocsLayout from '$docs/components/DocsLayout'
+import UsingStreamsViaApiContent from '$docs/content/streams/usingStreamsViaApi.mdx'
+
+const UsingStreamsViaApi = () => (
+    <DocsLayout>
+        <Helmet title="Work with streams via API | Streamr Docs" />
```

Comment:
Did you know about `titleTemplate`? https://github.com/nfl/react-helmet#reference-guide

Maybe could centralise this like:

```js
import React from 'react'
import { Helmet } from 'react-helmet'

export default function DocsHelmet({ titleTemplate = '% | Streamr Docs', ...props }) {
    return <Helmet titleTemplate={titleTemplate} {...props} />
}
```

Then it'd only need a single file change to change how we organise the title.

Reply:
That's not a bad idea! 👍